docs/command-reference.md

```diff
@@ -486,17 +520,217 @@ Arguments:
   -?|--help             Show help information.
 ```
 
+### neoxp contract validate
+
+The `neoxp contract validate` checks a given contract for compliance with proposal specification. It has two subcommands.
+
+#### nepxp contract validate nep11
+
+```
+Usage: neoxp contract validate nep11 [options] <ContractHash>
+
+Arguments:
+  ContractHash        Path to contract .nef file
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+This command checks if the specified contract is NEP-11 compliant.
+
+#### nepxp contract validate nep17
+
+```
+Usage: neoxp contract validate nep17 [options] <ContractHash>
+
+Arguments:
+  ContractHash        Path to contract .nef file
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+This command checks if the specified contract is NEP-17 compliant.
 
 ## neoxp show
 
-The `show` command will display information from the blockchain. There are multiple subcommands 
-representing the different  information that is available:
+The `show` command displays information from the blockchain. There are multiple subcommands 
+representing the different  information that is available.
+
+### neoxp show balance
+
+```
+Usage: neoxp show balance [options] <Asset> <Account>
+
+Arguments:
+  Asset               Asset to show balance of (symbol or script hash)
+  Account             Account to show asset balance for
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show balance` displays the balance of a single NEP-17 asset (including NEO and GAS) of a specific account.
+
+### neoxp show balances
+
+```
+Usage: neoxp show balances [options] <Account>
+
+Arguments:
+  Account             Account to show asset balances for
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show balances` displays the balance of all NEP-17 asset (including NEO and GAS) owned by a specific account.
+
+### neoxp show block
+
+```
+Usage: neoxp show block [options] <BlockHash>
+
+Arguments:
+  BlockHash           Optional block hash or index. Show most recent block if
+                      unspecified
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show block` displays the contents of a single block, specified by index or hash.
+
+### neoxp show nft
+
+```
+Usage: neoxp show nft [options] <Contract> <Account>
+
+Arguments:
+  Contract            NFT Contract (Symbol or Script Hash)
+  Account             Account to show NFT (Format: Script Hash, Address, Wallet name)
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show nft` displays the content of an NFT contract for a specified account.
+
```

Comment:
It is suggested to supplement:
The output will consist of TokenId in Base64 format and TokenId in big-endian Hex string format.